how to run this example ?

C:\code\fastify\fastify-example-twitter\frontend>npm build
npm WARN build `npm build` called with no arguments. Did you mean to `npm run-sc
ript build`?

in another terminal: 
> server@1.0.0 start C:\code\fastify\fastify-example-twitter
> fastify index.js "--log-level" "trace" "--port" "3001"

AsyncFunction
assert.js:212
assert.ifError = function ifError(err) { if (err) throw err; };
                                                  ^

Error: ENOENT: no such file or directory, stat 'C:\code\fastify\fastify-example-
twitter\frontend\build'
    at fs.statSync (fs.js:968:11)
    at checkPath (C:\code\fastify\fastify-example-twitter\node_modules\fastify-s
tatic\index.js:91:16)
    at checkPathsForErrors (C:\code\fastify\fastify-example-twitter\node_modules
\fastify-static\index.js:76:13)
    at fastifyStatic (C:\code\fastify\fastify-example-twitter\node_modules\fasti
fy-static\index.js:15:17)
    at Plugin.exec (C:\code\fastify\fastify-example-twitter\node_modules\avvio\p
lugin.js:33:17)
    at Boot.loadPlugin (C:\code\fastify\fastify-example-twitter\node_modules\avv
io\plugin.js:114:10)
    at Task.release (C:\code\fastify\fastify-example-twitter\node_modules\fastq\
queue.js:127:16)
    at worked (C:\code\fastify\fastify-example-twitter\node_modules\fastq\queue.
js:169:10)
    at toLoad.finish (C:\code\fastify\fastify-example-twitter\node_modules\avvio
\plugin.js:117:7)
    at done (C:\code\fastify\fastify-example-twitter\node_modules\avvio\plugin.j
s:74:7)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! server@1.0.0 start: `fastify index.js "--log-level" "trace" "--port" "3
001"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the server@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\samsom\AppData\Roaming\npm-cache\_logs\2018-01-06T05_40_47
_161Z-debug.log

--- a/README.md
+++ b/README.md
@@ -9,7 +9,7 @@ The aim of this project is to show how `fastify` can be used.
 ### Pseudo-production
 ```bash
 # build frontend
-cd frontend && npm build && cd ..
+cd frontend && npm run build && cd ..
 # start backend
 npm start -- --log-level trace --port 3001
 ```

